### DISCUSSION TITLE
Database doesn't keep settings after server reboot

### DISCUSSION THREAD

**Discussion by sobeservices** [alexjustesen/speedtest-tracker#1371] [Q&A]
Everytime I have to reboot my server I lose all the settings and everything goes back to defaults.

For example logging in. I have to use the default user and password.

I loose all previous cron jobs all results.

Any help would be appreciated.

**Comment by alexjustesen** [comment]
Please share your `docker-compose.yml` file or we can't really help.

**Reply by sobeservices** [reply]
It's empty

**Reply by alexjustesen** [reply]
Take a look at the install docs. Make sure You've followed them and have a basic understanding of Docker and containerization.
